client/src/components/LoginContainer.js

```diff
@@ -0,0 +1,57 @@
+import React, {useState} from 'react';
+import {Link} from "react-router-dom";
+import {getUser} from '../mockAPI';
+
+export default function LoginContainer() {
+
+    //Value of the username input box
+    const [userNameInput, setUserNameInput] = useState('');
+    //Value of the password input box
+    const [passwordInput, setPasswordInput] = useState('');
+
+    //Login authorization
+    const authorize = (e) => {
+        //prevents the browser from performing its default behavior when a form is submitted.
+        //prevent page from refreshing
+        e.preventDefault();
+
+        //Login authorization code
+        
+        if (userNameInput ===  getUser(userNameInput).username && passwordInput === getUser(userNameInput).password) {
```

Comment:
In future we'll need to look at handling all authentication securely on the backend, but this is just fine for now! 😄